### DISCUSSION TITLE
Google Workspace Chrome Policy "tool" / Wiki update

### DISCUSSION THREAD

**Discussion by mateuszATiwoca** [uBlockOrigin/uBlock-issues#3390] [Show and tell]
Hey,
what is the polite way to make changes to wiki? 
It does allow me to just make edit but I assume someone has to approve them later?

Anyhow, 
previously pushing a Policy via Google Workspace admin console to Chrome was a massive pain so I've made a tool for it after I've figured it out and someone even added it to wiki 🎉 
After "add" feature was introduced this all broke and now I finally had time to fix the tool thx to discussion [here](https://github.com/uBlockOrigin/uBlock-issues/discussions/1834)

Updated tool is [here](https://github.com/Landsil/apps_script--GoogleWorkspace-API/blob/master/uBlock_Origin_GSuite_policy.gs)

Process is much simpler now so arguably it's not as helpful but it may be worth putting it in the wiki again.
I hope it helps someone.

**Comment by krystian3w** [comment]
If it's a simple change of link to a newer one, you will find it in an editable version: https://github.com/uBlockOrigin/uBlock-issues/wiki <b><sup> [\[1\]][1] </sup></b>.

There must be some way to review it if some day dormant bot has wandered around the Wiki (That some human being estimates that the Wiki can be updated between repositories).

How you feel the need to write a few sentences depends on your mastery of English (Mateo written as 'Mateusz' sounds Slavic).

Slightly worse if you mean that the old URL from uBo wiki with the long description is now 404 (but it can be brought back to life by writing from scratch with the old title) &ndash; then paste there the link to the latest wiki entry due to the lack of [#redirect](https://en.wikipedia.org/wiki/Help:Redirect).

[1]: https://github.com/uBlockOrigin/uBlock-issues/discussions/3390#discussioncomment-11907332

**Reply by krystian3w** [reply]
It looks now needed contributor or even collaborator to edit dev wiki; seems thanks @gomoku multi accounts.
